.github/workflows/export_github_data.yml

```
@@ -4,6 +4,11 @@ on:
   schedule:
     - cron: "20 7 * * *"
 
+permissions:
+  id-token: write
+  contents: read
+  pull-requests: write
```

I think these permissions need to be adjusted like so:

```suggestion
  contents: read
  issues: read
  pull-requests: read
  security-events: read
```